source/change-streams/change-streams.rst

```diff
@@ -155,6 +169,16 @@ If an aggregate command with a ``$changeStream`` stage completes successfully, t
      */
     to: Optional<Document>;
 
+    /**
+     * Only present for ops of type 'rename', 'createIndexes', 'dropIndexes', 'shardCollection', 'reshardCollection', 'refineCollectionShardKey'.
```

Comment:
In reading the syntax document, it looks like `operationDescription` is also available for `modify` and `create`?

Reply:
Yeah, you're right.  I got this list from the section in the syntax document that says:

> operationDescription
> >  The specification of the event. In addition to the createIndexes, dropIndexes, shardCollection, reshardCollection,  refineCollectionShardKey and migrateLastChunkFromShard listed above, the existing rename event will also include this field. 

But I confirmed that `operationDescription` is present for `create` and `modify` as well.